client/components/projects/nav-bar.tsx

```diff
@@ -62,11 +55,17 @@ const UnderLine = styled.div`
 
 interface Props {
   projects: Types.Project[];
+  stories: Types.Story[];
 }
 
-const NavBar = ({ projects }: Props) => {
+const NavBar = ({ projects, stories }: Props) => {
   const [showStoryList, toggleStoryList] = useState(false);
 
+  const storyReviews = [];
+  stories.forEach(story => {
```

Comment:
Great use-case for `map`:

```typescript
const storyReviews = stories.map(story => (
  { storyName: story.name, estimateValue: null, storyID: story.id }
)); // TODO: get estimate value from the database migration to add estimates
```

... *Edit* This looks like it gets removed entirely in the next commit?

Reply:
Yes, I just added a commit to make sure we get the stories from the selected one instead of the static list Simon had before

Reply:
I appreciate the scaffolding 👍